packages/appium/lib/cli/setup-command.js

```diff
@@ -0,0 +1,163 @@
+import _ from 'lodash';
+import { SETUP_SUBCOMMAND } from '../constants';
+import {runExtensionCommand} from './extension';
+import { fs } from '@appium/support';
+
+/**
+ * Driver names listed in KNOWN_DRIVERS to install by default.
+ */
+export const MOBILE_DRIVERS = ['uiautomator2', 'espresso', 'xcuitest'];
+export const SUBCOMMAND_MOBILE = 'mobile';
+
+
+/**
+ * Driver names listed in KNOWN_DRIVERS to install by default.
+ */
+export const DESKTOP_APP_DRIVERS = ['mac2'];
+export const SUBCOMMAND_DESKTOP = 'desktop';
+
+/**
+ * Driver names listed in KNOWN_DRIVERS to install by default.
+ */
+export const BROWSER_DRIVERS = ['uiautomator2', 'xcuitest', 'chromium', 'safari', 'gecko'];
```

Comment:
I assume we don't need to install 'uiautomator2', 'xcuitest' in such case as this is only about desktop browser. Also safari should be only installed for macOS hosts

Reply:
I agree - while mobile browsers do need uia2 and xcuitest (although Safari driver does work on iOS too, hm), it's likely safe to assume that the end user would expect the command to be only for desktop browsers. Would suggest also adjusting the help message to clarify this.